src/svg.js

```diff
@@ -55,7 +55,7 @@ Canvas.init = elmOrId => {
     svg = createElm('svg');
     elm.appendChild(svg);
   }
-  attrs(svg, {width: '100%', height: '100%'});
+  attrs(svg, {width: _width, height: _height});
```

Comment:
I think that error when you use `width` instead of `_width` is because it wants you to write:
`attrs(svg, {width, height });`

If you use the same name for the key and the value in an object you don't need a `:`

Reply:
Got it, Thanks